app/services/default_values_object.rb

```diff
@@ -0,0 +1,21 @@
+class DefaultValuesObject
```

Comment:
Yeah, the more I'm reading about it, Im seeing that whatever operations you are doing on an objects data (in this case, on the data utilized by the forms), you should define those methods where the data needs to be. In this case, would it also be useful to define those defaults in the form object also, instead of in the controller?